.github/workflows/ci.yaml

```diff
@@ -21,11 +21,18 @@ jobs:
         uses: dtolnay/rust-toolchain@stable
         with:
           toolchain: stable
+          targets: wasm32-unknown-unknown
 
       - name: Run cargo check
-        uses: dtolnay/rust-toolchain@stable
-        with:
-          command: check
+        run: cargo check
+
+      - name: Run cargo check for math with no-std
+        run: cargo check --package lambdaworks-math --no-default-features
+
+      - name: Run cargo build ensure-no_std crate
+        run: |
+          cd ensure-no_std
+          cargo build
```

Comment:
Does the workflow keep the current dir? In that case this should include a `cd ..` or move this step to the end of it.

Reply:
Yes it does